### PR TITLE
Update list.txt

### PR DESCRIPTION
**Pull request by Arbyy**

### CODE REVIEW

File: list.txt [hunk 1]
```diff
@@ -18,3 +18,7 @@
 *://issueexplorer.com/*
 *://uonfu.com/*
 *://gitcodeask.cc/*
+*://grabthiscode.com/*
+*://*.hotexamples.com/*
+*://hotexamples.com/*
```
Comment (carlalbrecht) [inline list.txt]: afaict this is redundant when you have a wildcard subdomain rule

**Review by carlalbrecht — APPROVED** [review]
overall, nothing sus

File: list.txt (hunk 1 above)
Comment (Arbyy) [reply list.txt]: I think its just regex right? like `*://*.hotexamples.com/*` != `*://hotexamples.com/*` in regex at least

Comment (Arbyy) [reply list.txt]: forget that https://iorate.github.io/ublacklist/docs/advanced-features

Comment (carlalbrecht) [reply list.txt]: you just got pranked :^)

**Pull request merged**